api/src/http/routes/change-password.ts

```diff
@@ -74,13 +74,13 @@ export async function changePassword(server: FastifyInstance) {
 
         if (newPasswordMatch) {
             return reply.status(400).send({
-                message: "A nova senha não pode ser igual à antiga senha",
+                message: "A nova senha não pode ser igual à antiga senha.",
```

Comment:
```suggestion
                message: "A nova senha não pode ser igual à senha antiga.",
```

Estou sugerindo essa mudança pq fica uma mensagem menos robótica.